tutorials/2d/2d_movement.rst

```diff
@@ -241,33 +182,23 @@ is set by the mouse position instead of the keyboard. The character will always
 
     public class Movement : CharacterBody2D
     {
-        [Export] public int speed = 200;
-
-        public Vector2 velocity = new Vector2();
+        [Export] public int speed = 400;
 
         public void GetInput()
         {
             LookAt(GetGlobalMousePosition());
-            velocity = new Vector2();
-
-            if (Input.IsActionPressed("down"))
-                velocity = new Vector2(-speed, 0).Rotated(Rotation);
-
-            if (Input.IsActionPressed("up"))
-                velocity = new Vector2(speed, 0).Rotated(Rotation);
-
-            velocity = velocity.Normalized() * speed;
+            Velocity = Transform.x * Input.get_axis("down", "up") * speed
         }
 
         public override void _PhysicsProcess(float delta)
```

Comment:
In 4.0, the `delta` parameter of `_Process` and `_PhysicsProcess` is now a 64-bit type (like in GDScript).
```suggestion
        public override void _PhysicsProcess(double delta)
```